specification/app/resource-manager/Microsoft.App/preview/2023-05-02-preview/Subscriptions.json

```diff
@@ -15,13 +15,13 @@
     "application/json"
   ],
   "paths": {
-    "/subscriptions/{subscriptionId}/providers/Microsoft.App/getVerificationId": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.App/getCustomDomainVerificationId": {
```

Comment:
>getCustomDomainVerificationId [](http://example.com/codeflow?start=61&length=29)

Isn't this a breaking change for your customers? Are there any customers who are using this API?

Reply:
The target preview version has not been released yet.